src/lib/djerba/plugins/supplement/body/resources/TAR.description.versions.html

```diff
@@ -18,7 +18,7 @@
 against reference genome <a href=${versions.REFERENCE_GENOME_LINK}>${versions.REFERENCE_GENOME_VERSION}</a>. 
 Unique molecule based error suppression was completed with <a href=${versions.CONSENSUSCRUNCHER_LINK}>ConsensusCruncher</a>. 
 SNVs and INDELs were called using <a href=${versions.MUTECT2_LINK}>MuTect2</a> (GATK ${versions.MUTECT2_VERSION}).
-Twelve genes were annotated with <a href=${versions.VARIANTEFFECTPREDICTOR_LINK}>VariantEffectPredictor</a> (${versions.VARIANTEFFECTPREDICTOR_VERSION}) 
+Fourteen genes were annotated with <a href=${versions.VARIANTEFFECTPREDICTOR_LINK}>VariantEffectPredictor</a> (${versions.VARIANTEFFECTPREDICTOR_VERSION}) 
```

Comment:
Should this be 15 genes? We have added 3 genes to the previous 12, right?

Reply:
12 + 3 (NF1, VHL, CDH1) - 1 (ABCB1)  = 14 (I also removed ABCB1)

Reply:
Oh, I see. All good then. :grin: